lib/tasks/remove_dickinson_placeholder_thumbnails.rake

```diff
@@ -0,0 +1,28 @@
+namespace :hyku do
+  desc "Remove Hyku Commons placeholder file sets from tenant"
+  task remove_placeholder_file_sets: :environment do
+    # The original ask was for the Dickinson tenant, but passing in an ENV var would be useful for testing
+    # example:
+    # ```sh
+    # REMOVE_PLACEHOLDER_FILE_SETS_FROM="dev.hyku.test" rake hyku:remove_placeholder_file_sets
+    # ```
+    account_cname = ENV['REMOVE_PLACEHOLDER_FILE_SETS_FROM'] || "dickinson.hykucommons.org"
+
+    Account.find_each do |account|
+      next unless account.cname == account_cname
+
+      begin
+        switch!(account.cname)
```

Comment:
A few more thoughts: 

This works but after going through QA it was kind of a pain having to remember to set the env var to test what I needed.

I think it would be nice if we could pass the cname when calling the rake task command. I also think the file name is a misnomer if we want this to be more dynamic.

Reply:
I haven't tested this but maybe something like this? 

```ruby

# rename file to lib/tasks/remove_placeholders.rake

namespace :hyku do
  desc "Remove Hyku Commons placeholder file sets from tenants"
  task :remove_placeholder_file_sets, [:cnames] => :environment do |t, args|
    cnames = args[:cnames].split(',')
    cnames.each do |cname|
      account = Account.find_by(cname: cname)
      next unless account

      begin
        switch!(account.cname)
        puts "********************** switched to #{account.cname} **********************"

        file_set_titles = ["HykuCommonsPlaceholder.pdf", "hykuplaceholderarchives.pdf"]
        query = file_set_titles.map { |title| "title_tesim:\"#{title}\"" }.join(" OR ")
        file_sets = FileSet.where(query)
        file_sets.each { |file_set| file_set.destroy(eradicate: true) }
      rescue StandardError => e
        puts "********************** error: #{e} **********************"
      end
    end
  end
end

```